composer.json

```diff
@@ -10,12 +10,25 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "ezsystems/ezpublish-legacy": ">=2014.11"
+        "ezsystems/ezpublish-legacy": ">=2014.11",
+        "ezsystems/ezpublish-kernel": "dev-master"
```

Comment:
Yeah, i considered it... but technically, it *is* required, not only in dev (since it uses interfaces/classes from there (need to double check that, maybe only tests do)).